packages/aws-cdk-lib/core/lib/duration.ts

```diff
@@ -62,6 +62,9 @@ export class Duration {
   /**
    * Parse a period formatted according to the ISO 8601 standard
    *
+   * Days are the largest ISO duration supported, i.e.,
+   * months and years are not supported.
+   *
```

Comment:
It might be worth adding an example:

```suggestion
   *
   * @example 'P1DT2H3M4.567S' would represent 1 day, 2 hours, 3 minutes, 4 seconds, and 567 milliseconds.
```